samples/model-builder/model_registry/noxfile.py

```diff
@@ -0,0 +1,311 @@
+# Copyright 2019 Google LLC
```

Comment:
I don't have full context of what this file is doing, I'll leave someone else to review.

Reply:
It is a config file for code tests